### DISCUSSION TITLE
problem in connecting to my old github account

### DISCUSSION THREAD

**Discussion by Ranimto** [community/community#131976] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

i forget my email that is connected to my other github account but i have the password and the username correct but i can t verify it via email how can i do to get my OLD account 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by danialtavakoli** [comment]
If you have forgotten the email address associated with your GitHub account but still remember your username and password, there are several steps you can take to recover your account.

First, try logging in with your username and password. Sometimes, GitHub provides hints or prompts that can help you remember the associated email address. Next, go through your email accounts to find any emails from GitHub. Look for registration emails, notifications, or any communication from GitHub that might help identify the associated email address.

If you can't recover the email address yourself, you can contact GitHub Support for assistance. Provide them with your username and any other relevant information to verify your identity. To contact GitHub Support, go to the [GitHub Support page](https://support.github.com/), select "Contact GitHub Support," choose the appropriate category (e.g., "Account Recovery"), and fill out the form with as much detail as possible.

Additionally, if you have set up SSH keys or personal access tokens for the account, you might be able to use them to verify your identity or gain access to your account. Check if your browser has saved the login credentials for the GitHub account, which might include the email address. You can also check any third-party applications or services where you've used GitHub for authentication, as these services sometimes list the associated email address.

If all else fails and you are unable to recover your email through these methods, GitHub Support is your best bet for regaining access to your account. They are usually very responsive and can help you recover your account if you provide enough information to verify your identity.

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @Ranimto , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
